go.mod

```diff
@@ -1,14 +1,36 @@
 module github.com/grpc-ecosystem/grpcdebug
 
-go 1.15
+go 1.22.7
 
 require (
-	github.com/dustin/go-humanize v1.0.0
-	github.com/envoyproxy/go-control-plane v0.10.3
-	github.com/golang/protobuf v1.5.2
+	github.com/dustin/go-humanize v1.0.1
```

Comment:
Can we merge the require list and minimize it?

Reply:
Are you referring to the indirect require list below? That is from the result of `go mod tidy` (probably in newer go version) where it explicitly states the indirect dependencies to make the build more stable.

I can try remove the files and rerun `go mod init` to generate a new one (instead of patching it, which is what I am doing now)